### DISCUSSION TITLE
DateTime2: Intended Functionality of "Today" Button in DateInput2?

### DISCUSSION THREAD

**Discussion by scaper12123** [palantir/blueprint#6164] [Q&A]
How is the "Today" button in the DateInput2 component intended to work when considering timezones?

For example, say I'm operating out of EST and I want to create a component that will only think in terms of UTC. Given that the DateInput2 component will assume it is operating on my local timezone by default, my assumption is that programmatically setting it to work in UTC instead will cause the "Today" button to adjust the input time according to the difference between EST and UTC. Is this true? 

If not, then how would I create a setup such that, regardless of what time zone I'm working in, inputting "Today" will give me the appropriate UTC timezone?

**Comment by swar-mukh** [comment]
On my system with `"@blueprintjs/datetime2": "^0.9.27"`, the timezone rather strangely somehow defaults to `845` rather than `530` which is where I am based out. This is causing real headaches in parsing.
